lumiere_beta.py

```diff
@@ -1972,28 +1892,20 @@ def draw(self, context):
                                         row = col.row(align=True)
                                         if cobj.expanded and cobj.show == False:
                                             row.alert = True
-                                        row.operator("object.list_populate", icon='BLANK1').act_light = cobj.name
+                                        row.operator("object.add_gradient", icon='BLANK1').act_light = cobj.name
                                         row.alert = False
                                     else:
                                         col = box.column(align=True)
                                         row = col.row(align=True)
                                         row.prop(cobj, "gradient", text='Gradient type :', icon='BLANK1')
-                                        row = col.row(align=True)
+                                        row = box.row(align=True)
                                         
                                         if cobj.typlight in ("Panel", "Pencil"): row.prop(cobj, "typgradient", text="")
-                                        row.prop(cobj, "gradinterpo")
 
-                                        rows = 2
-                                        row = col.row(align=True)
-                                        row.template_list("SCENE_UL_list", "", cobj.prop_group, "coll", cobj.prop_group, "index", rows=rows)
-
-                                        col = row.column(align=True)
-                                        op = col.operator("custom.list_action", icon='ZOOMIN', text="")
-                                        op.action = 'ADD'
-                                        op.act_light = cobj.name
-                                        op = col.operator("custom.list_action", icon='ZOOMOUT', text="")
-                                        op.action = 'REMOVE'
-                                        op.act_light = cobj.name
+                                        #---Template Color Ramp
+                                        #---Accesses the color ramp from the material
+                                        colramp = cobj.data.materials['Mat_' + cobj.name].node_tree.nodes['ColorRamp']
```

Comment:
Instead of cobj.name, you have to use cobj.data.name

So the user can change the name in the panel and we can keep track of the real name in the data properties